src/daemon/rpmostreed-os.c

```diff
@@ -550,6 +550,27 @@ deploy_flags_from_options (GVariant *options,
   return ret;
 }
 
+static gint*
+get_sparse_fd_array_from_idxs (gint     *fds,
```

Comment:
I stared at this for a few minutes and for some reason it's confusing me still; I think it's that the return value is *not* sparse?